README.md

```diff
@@ -0,0 +1,76 @@
+# AppleTun
+
+## Description
+
+A utility for creating and connecting to a local VPN for ios devices allowing HTTP proxy without device supervision for all connections (not limited to Wi-Fi)
+
+## Installation
+
+```shell
+python3 -m pip install -U AppleTun
+```
+
+## Requirements
+
+[StrongSwan](https://github.com/strongswan/strongswan) installation is required.
+Installation of StrongSwan can be done via `brew install strongswan`, any other package manager, or a manual build (incase special flags are required, exporting of session keys to files for example).
+
+Python requirements are stated in `requirements.txt`.
```

Comment:
You already have a proper python package, so you don't need to tell users about using the low-level `requirements.txt`. I do recommend you mention the `strongwan` requirement above the installation since this is a prequisite.